Request/Request.php

```diff
@@ -49,18 +50,42 @@ final class Request
      */
     public function __construct(HttpRequest $request)
     {
-        $this->pageNum = (int)$request->query->get('page', 1);
-        $this->limit = (int)$request->query->get('limit', -1);
-        $this->query = $request->query->get('filter');
-        if ($this->query !== null && !is_array($this->query)) {
+        try {
+            $this->pageNum = (int)$request->query->get('page', 1);
+        } catch (BadRequestException) {
```

Comment:
Hm... Are you sure `$request->query->get` can return BadRequestException?

Reply:
It does throw one https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/HttpFoundation/InputBag.php#L37

The tests should also prove this since it returns the correct exception

Reply:
Aha, ok. Thank you for this change.